src/adapters/github.test.ts

```diff
@@ -11,6 +11,10 @@ const mockMigrationContext = () =>
       spec: {
         id: 'test-migration',
         title: 'Test migration',
```

Comment:
nit: The value is used for the commit message and so we should use semantic commit convention (e.g., feat, docs, etc...)

Reply:
I think the issue message doesn't need to have semantic convention.

Reply:
Ah, I got my wires crossed 🥹